### ISSUE TITLE
1.5.2 build error

### ISSUE THREAD

**Issue by Stallos11** [LeandroSQ/android-ble-made-easy#11]
I am unable to use the 1.5.2 update since the build has an error.

https://jitpack.io/com/github/LeandroSQ/android-ble-made-easy/v1.5.2/build.log

if someone encounters the error `Failed to resolve: com.github.LeandroSQ:android-ble-made-easy:v1.5.2`
Just use the 1.5.1 update using 
```
dependencies {
    // ...
    implementation 'com.github.LeandroSQ:android-ble-made-easy:v1.5.1'
}
```

**Comment by KrasimirGatevMentorMate** [comment]
Same here

**Comment by LeandroSQ** [comment]
Hi folks, thanks for pointing this out!

This issue should be fixed with [v1.6.0](https://jitpack.io/com/github/LeandroSQ/android-ble-made-easy/1.6.0/build.log)
